### PR TITLE
More eris sec ports

### PR DESCRIPTION
**Pull request by cdb-is-not-good**
New webbed armor vests for Marshal officers/Spec, as well as fresh drip in the form of combat turtlenecks.
Adds the Seintmetall "Ironhammer"(har har) breaching hammer, a fairly robust breaching tool for the Marshals creatable via their disk.

## About The Pull Request
<details>
<summary>
	About The Pull Request
</summary>
<hr>

	
<hr>
</details>

## Changelog
:cl:

/:cl:

### CODE REVIEW

File: code/datums/autolathe/tools.dm [hunk 1]
```diff
@@ -98,6 +98,10 @@
 	name = "greyson sledgehammer"
 	build_path = /obj/item/tool/hammer/powered_hammer/onestar_hammer
 
+/datum/design/autolathe/tool/ironhammer
+	name = "name thing"
```
Comment (Trilbyspaceclone) [inline code/datums/autolathe/tools.dm:102]:
```suggestion
	name = "GmbH \"Ironhammer\" Breaching Hammer"
```

File: code/game/objects/items/weapons/tools/hammer.dm [hunk 2]
```diff
@@ -110,6 +110,21 @@
 	max_upgrades = 5
 	price_tag = 15
 
+/obj/item/tool/hammer/IH
```
Comment (r4d6) [inline code/game/objects/items/weapons/tools/hammer.dm:113]:
```suggestion
/obj/item/tool/hammer/ironhammer
```

File: code/datums/autolathe/tools.dm [hunk 3]
```diff
@@ -98,6 +98,10 @@
 	name = "greyson sledgehammer"
 	build_path = /obj/item/tool/hammer/powered_hammer/onestar_hammer
 
+/datum/design/autolathe/tool/ironhammer
+	name = "GmbH \"Ironhammer\" Breaching Hammer"
+	build_path = /obj/item/tool/hammer/IH
```
Comment (r4d6) [inline code/datums/autolathe/tools.dm:103]:
```suggestion
	build_path = /obj/item/tool/hammer/ironhammer
```

**Review by r4d6 — CHANGES_REQUESTED** [review]
It's better to not have the path be 2 capital letter, as that is generally the kind of thing used for local variables within a proc.

**Review by Trilbyspaceclone — APPROVED** [review]
Good First PR

**Comment by cdb-is-not-good** [comment]
> Good First PR

Die

**Pull request merged**